components/console-backend-service/internal/domain/k8s/limitrange_resolver.go

```diff
@@ -31,3 +31,9 @@ func (lr *limitRangeResolver) LimitRangesQuery(ctx context.Context, namespace st
 
 	return lr.converter.ToGQLs(limitRange), nil
 }
+
+func (lr *limitRangeResolver) CreateLimitRange(ctx context.Context, namespace string, name string, limitRange gqlschema.LimitRangeInput) (*gqlschema.LimitRange, error) {
+	createdLimitRange, err := lr.lister.Create(namespace, name, limitRange)
+
```

Comment:
``` go
	if err != nil {
		glog.Error(errors.Wrapf(err, "while creating %s `%s` in namespace `%s`", pretty.LimitRange, name, namespace))
		return nil, gqlerror.New(err, pretty.LimitRange, gqlerror.WithName(name), gqlerror.WithNamespace(namespace))
	}
```

If we haven't pretty for LimitRange, please add it to relevant file, and use it above.

Reply:
done